doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -652,6 +652,26 @@ The following changes are recommended for your application to work optimally aft
 
 * For LwM2M applications, replace the :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_DTLS_CID` Kconfig option with :kconfig:option:`CONFIG_LWM2M_DTLS_CID`.
 
+* For Mesh applications, the Sensor API has been re-written, and the previous Sensor API is to be depreciated.
+  A new Sensor API has been created that should be used for new applications and applications that will be continuously updated.
+  How the new Sensor API is working is shown in the :ref:`bluetooth_mesh_sensor_client`, :ref:`bluetooth_mesh_sensor_server` and :ref:`bluetooth_mesh_light_lc` samples, as they all have been updated to use the new Sensor API.
```

Comment:
I don't know if we should do this now or after the docs are updated, but `bt_mesh_sensors_readme` should also be linked here since that contains the detailed description of how all the API pieces fit together (or, it will do, once it's updated)